qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -159,9 +160,17 @@ def run(self,
                 DeprecationWarning,
                 stacklevel=3)
 
+        # Add default OpenMP options
+        gpu = backend_options is not None and 'gpu' in backend_options.get('method', '')
+        profiled_options = get_performance_options(gpu)
```

Comment:
This should be handled as pat of the format qobj, and only add the profile option to the qobj config if they aren't present in options or run options